### PR TITLE
Onboarding AM Role Assignment Batch service.

### PR DESCRIPTION
**Pull request by nitishsingla1234**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-796

### Change description ###
Onboarding AM Role Assignment Batch service. This service will spin up the Kubernetes pods at specified times and perform the required actions.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by nitishsingla1234** [comment]
Hi, Can someone please have a look a this PR.
Thanks

File: CODEOWNERS [hunk 1]
```diff
@@ -226,6 +226,8 @@ k8s/aat/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/perftest/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/aat/common/am/am-role-assignment-service.yaml @hmcts/am-devs
 k8s/aat/common/am/am-judicial-booking-service.yaml @hmcts/am-devs
+k8s/aat/cluster-00/am/role-assignment-batch.yaml @hmcts/am-devs
```
Comment (adusumillipraveen) [inline CODEOWNERS:229]:
```suggestion
k8s/aat/cluster-**/am/role-assignment-batch.yaml @hmcts/am-devs
```

**Comment by nitishsingla1234** [comment]
Hi @adusumillipraveen After the code update, we have lost the approval. Can you please approve it.
Thanks

File: CODEOWNERS [hunk 2]
```diff
@@ -226,6 +226,8 @@ k8s/aat/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/perftest/common/am/accessmgmt-api.yaml @hmcts/am-devs
 k8s/aat/common/am/am-role-assignment-service.yaml @hmcts/am-devs
 k8s/aat/common/am/am-judicial-booking-service.yaml @hmcts/am-devs
+k8s/aat/cluster-**/am/role-assignment-batch.yaml @hmcts/am-devs
+k8s/aat/cluster-01/am/role-assignment-batch.yaml @hmcts/am-devs
```
Comment (adusumillipraveen) [inline CODEOWNERS:230]:
```suggestion
```

**Pull request merged**